### PR TITLE
fix: Introduction of `devices.update`

### PR DESCRIPTION
**Pull request by andrii-balitskyi**

### CODE REVIEW

**Review by codetheweb — COMMENTED** [review]
Looks pretty good!

I think as far as types go we should have some common properties type that `Device` extends from (`interface Device<Properties extends CommonDeviceProperties, Type = DeviceType>`) and then specify mutable properties as a subset of that (`type MutableDeviceProperties = Pick<CommonDeviceProperties, 'name'>`).

File: src/types/route-requests.ts [hunk 1]
```diff
@@ -26,3 +26,10 @@ export type AccessCodeDeleteRequest = {
   device_id?: string
   access_code_id: string
 }
+
+export type DeviceUpdateRequest = {
+  device_id: string
+  properties?: DeviceProperties
```
Comment (seveibar) [inline src/types/route-requests.ts:32]: Hmm this is tricky, the properties depend on the device type, maybe we could create a type BaseDeviceProperties and have LockProperties extend it

**Comment by codetheweb** [comment]
@andrii-balitskyi if you update your local Git remote with `git remote remove origin` and `git remote add origin git@github.com:seamapi/seamapi-javascript.git` (replace with the HTTPS URL if that's what you use) and run `yarn build:docs` to fix the org name references I think this will be good to merge. 👍

**Comment by andrii-balitskyi** [comment]
@codetheweb now I have some other issue, u changed the types? 
image

**Comment by codetheweb** [comment]
Yeah, sorry about that. Take a look at `devices.get()` above that to see what you need to change. You'll have to add a `DeviceUpdateResponse` interface.

**Review by seveibar — APPROVED** [review]

**Pull request merged**